chapters/hlsl.adoc

```diff
@@ -7,6 +7,7 @@ ifndef::images[:images: images/]
 
 [[hlsl-in-vulkan]]
 = HLSL in Vulkan
+:toc:
```

Comment:
Any reason this was explicitly added? Antora already takes care o the toc, this might lead to a duplicated toc in the docs site build.